package.json

```diff
@@ -35,13 +36,24 @@
     "chai": "^4.2.0",
     "eslint": "^7.7.0",
     "eslint-config-eslint": "^6.0.0",
+    "eslint-plugin-jsdoc": "^22.1.0",
+    "eslint-plugin-node": "^11.1.0",
     "eslint-release": "^3.1.0",
     "mocha": "^8.1.1"
   },
   "dependencies": {
     "ajv": "^6.12.4",
     "debug": "^4.1.1",
+    "espree": "^7.3.0",
```

Comment:
This should be probably either always in sync, or resolved from ESLint instead.

Reply:
Resolving this from ESLint seems like a better way to go :+1:

Reply:
I'd say we can do this in the short-term, but in the long-term this should be passed in from ESLint. (Right now, this code isn't actually being used in ESLint, so I don't think it should cause issues either way.)